apps/s_server.c

```diff
@@ -1694,6 +1703,11 @@ int s_server_main(int argc, char *argv[])
 #endif
 
 #ifndef OPENSSL_NO_KTLS
+    if (use_sendfile && enable_ktls == 0) {
```

Comment:
Would it make send for `use_sendfile` to set `enable_ktls` true instead of erroring?  Perhaps with a warning?

Reply:
A warning might be better. Automatically setting `enable_ktls` to true may confuse the user